src/guiFormSpecMenu.cpp

```diff
@@ -2217,15 +2320,15 @@ void GUIFormSpecMenu::drawList(const ListDrawSpec &s, int phase,
 	video::IVideoDriver* driver = Environment->getVideoDriver();
 
 	Inventory *inv = m_invmgr->getInventory(s.inventoryloc);
-	if(!inv){
+	if (!inv){
```

Comment:
Well, thanks for correcting the code style but either fix the whole line or un-touch this code.
`if (condition) {` Brace on the same, with spaces around the outer paranteses.